Packs/FortiAuthenticator/ReleaseNotes/1_0_1.md

```diff
@@ -2,4 +2,7 @@
 ##### FortiAuthenticator
 - Fixed an issue where the ***fortiauthenticator-get-user*** command failed on LDAP user searches.
 - Added the arguments **username* and **token_serial** to the following commands: ***fortiauthenticator-update-user*** and ***fortiauthenticator-update-user***.
+<<<<<<< HEAD
+=======
 - Upgraded the Docker image to: *demisto/python3:3.9.6.24019*.
+>>>>>>> ca5de323bb1502d83a2f9b8e331d03d8f7bfb75d
```

Comment:
Please drop this file from the PR
as the update release notes are the 1_0_4.md
and 1_0_1.md shouldn't be touched